composer.json

```diff
@@ -0,0 +1,53 @@
+{
+    "require": {
+        "PHP": "^7.1",
+        "aws/aws-sdk-php": "^3.59",
+        "keboola/php-component": "^4.1"
+    },
+    "require-dev": {
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "keboola/coding-standard": "^4.0",
+        "keboola/php-temp": "^1.0",
+        "phpstan/phpstan-shim": "^0.9.2",
+        "phpunit/phpunit": "^7.0",
+        "symfony/process": "^4.0",
+        "keboola/datadir-tests": "^1.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Keboola\\S3Writer\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Keboola\\S3Writer\\Tests\\": "tests/phpunit/"
+        }
+    },
+    "scripts": {
+        "tests-phpunit": "phpunit",
+        "tests-datadir": "datadir-tests",
+        "tests": [
+            "@tests-phpunit"
```

Comment:
nepouští se datadir-tests (což může a nemusí být problém)

Reply:
Ale tím pádem nemusí být definované ani...

Reply:
nemám u writeru jak otestovat data, takže to celý mažu

Reply:
Už jsme řešili v https://github.com/keboola/datadir-tests/issues/9#issuecomment-394289163 že by bylo cool aby to výhledově umělo...